### PR TITLE
this my lesson one and two task one

### PR DESCRIPTION
**Pull request by arinzeatusiuba**
I am done with my task 1 for lesson 1 and 2 , please review @ChidiPatrick @promise-J  @OgJayKay
@Pieta9221 @Minihack @Melumm
please kindly note that our instructor asked that I avoid the lint error check

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Brandings</a></li>
+                <li><a href="#">Showreel</a></li>
+                <li><a href="#">Testimonial</a></li>
+                <li><a href="#">Contact Us</a></li>
+            </ul>
+        </nav>
+        <section>
+            <h3>"Motto: Zenith of Creativity"</h3>
```
Comment (Melumm) [inline html/index.html]: HI pal, the instruction says, "Within the <section> element, add an <h1> element with a main heading ".

**Review by Melumm — COMMENTED** [review]
Good work!!

File: html/index.html (hunk 1 above)
Comment (arinzeatusiuba) [reply html/index.html]: Thank you so much sir Melumm, I have corrected the element , Please check my code @Melumm

**Comment by Pieta9221** [comment]
Nice one 
Go ahead and merge

**Review by ChidiPatrick — COMMENTED** [review]
Great job! You can merge

**Pull request merged**